### PR TITLE
Detailed error message for authentication

### PR DESCRIPTION
**Pull request by Warhedgehog**
Displaying detailed error message when firebase refuse authentication

### CODE REVIEW

File: app/src/androidTest/java/sdp/moneyrun/authentication/LoginInstrumentedTest.java [hunk 1]
```diff
@@ -85,6 +88,28 @@ public void describeTo(Description description) {
         };
     }
 
+    public class ToastMatcher extends TypeSafeMatcher<Root> {
+
+        @Override
+        public void describeTo(Description description) {
+            description.appendText("is toast");
```
Comment (Tesa27) [inline app/src/androidTest/java/sdp/moneyrun/authentication/LoginInstrumentedTest.java:95]: Why is this necessary?

**Review by gauthierboeshertz — APPROVED** [review]

File: app/src/androidTest/java/sdp/moneyrun/authentication/LoginInstrumentedTest.java (hunk 1 above)
Comment (Warhedgehog) [reply app/src/androidTest/java/sdp/moneyrun/authentication/LoginInstrumentedTest.java:95]: That particularly I don't know. But the whole ToastMatcher is in the current implementation of tests useless

Comment (Tesa27) [reply app/src/androidTest/java/sdp/moneyrun/authentication/LoginInstrumentedTest.java]: alright!

**Review by Tesa27 — APPROVED** [review]
